docs/podman.1.md

```diff
@@ -137,7 +137,7 @@ the exit codes follow the `chroot` standard, see below:
 | [podman-exec(1)](podman-exec.1.md)        | Execute a command in a running container.                                      |
 | [podman-export(1)](podman-export.1.md)    | Export a container's filesystem contents as a tar archive.                     |
 | [podman-generate(1)](podman-generate.1.md)| Generate structured data based for a containers and pods.                      |
-| [podman-help(1)](podman-history.1.md)     | Show help information on podman.                                               |
+| [podman(1)](podman.1.md)                  | Show help information on podman.                                               |
```

Comment:
Alphabetize

Reply:
This one bothers me. Should I just remove it? I mean, the user is already reading `podman(1)`.

Reply:
Yes